app/utils/confirmRegistration/confirmRegistration.factory.js

```diff
@@ -0,0 +1,58 @@
+(() => {
+
+    'use strict';
+
+    angular
+        .module('app')
+        .factory('ConfirmRegistrationFactory', ConfirmRegistrationFactory);
+
+    ConfirmRegistrationFactory.$inject = ['$resource'];
+
+    function ConfirmRegistrationFactory($resource) {
+        return {
+            /* Factory must always return object */
+
+            checkEmail: function () { /* Its test function, for simulate server response */
```

Comment:
Please, remove this test function and use $resource object, which you commented at the bottom of this file.